GitFyle.Core.Api/Brokers/Loggings/LoggingBroker.cs

```diff
@@ -22,6 +22,11 @@ public class LoggingBroker : ILoggingBroker
 
         public void LogDebug(string message) =>
             this.logger.LogDebug(message);
+
+        public void LogWarning(string message) =>
+            this.logger.LogWarning(message);
+
+        public void LogCritical(Exception exception) =>
+             this.logger.LogCritical(exception, exception.Message);
```

Comment:
Remove this one.   We need [#31](https://github.com/The-Standard-Organization/GitFyle.Core.Api/issues/31) next and then LogCritical

Reply:
@cjdutoit ok